requirements-full.txt

```diff
@@ -31,8 +30,7 @@ five.localsitemanager==3.4
 funcsigs==1.0.2
 future==0.18.2
 ipaddress==1.0.23
-jeepney==0.7.1; python_version == '3.6'
-jeepney==0.8.0; python_version > '3.6'
+jeepney==0.8.0
 mock==4.0.3
```

Comment:
Did we not want to replace `mock` by `unittest.mock`?

Reply:
Zope itself does not use `mock` at all (according to GitHub search), but our dependencies might still use it.